src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball randBall = lottery.getRandomBall();
+            System.out.println(randBall);
```

Comment:
Let's inline the variable.
```suggestion
            System.out.println(...);
```